client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesClient.java

```diff
@@ -23,9 +23,12 @@
 import org.elasticsearch.action.ActionListener;
 import org.elasticsearch.action.admin.indices.delete.DeleteIndexRequest;
 import org.elasticsearch.action.admin.indices.delete.DeleteIndexResponse;
+import org.elasticsearch.action.admin.indices.exists.indices.IndicesExistsRequest;
+import org.elasticsearch.action.admin.indices.exists.indices.IndicesExistsResponse;
 
 import java.io.IOException;
-import java.util.Collections;
+
+import static java.util.Collections.emptySet;
```

Comment:
nit: can you revert the addition of the static import? It affects also other methods that you are not changing and that causes some noise in the PR.

Reply:
Will do 👍